server/src/apiClient/YelpClient.js

```diff
@@ -0,0 +1,26 @@
+import got from "got"
+import config from "../config.js"
+
+
+const yelpApiKey = config.yelpApiKey.secret
+
+class YelpClient {
+  static async getPlaces() {
```

Comment:
now that we have this working, we can pass arguments to the getPlaces function.. so that....

Reply:
This will also be filled in by the create-client-form branch